Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -48,17 +48,87 @@ public float SliderValue
             }
         }
 
+        [Header("Slider Axis Visuals")]
+
+        [Tooltip("The gameObject that contains the thumb visual. This will get rotated to match the slider axis")]
+        [SerializeField]
+        private GameObject thumbVisual = null;
+        public GameObject ThumbVisual
```

Comment:
Please add summary tags for these new public properties. They can match the above Tooltip text

Reply:
Done